lib/streamlit/connections/snowflake_connection.py

```diff
@@ -151,7 +151,7 @@ def query(
 
         Returns
         -------
-        pd.DataFrame
+        pandas.DataFrame
```

Comment:
nit: there might be also some other uses of `pd` in docstring scatter around the codebase.

Reply:
I thought I did a review to get rid of all of them previously but I can't find the PR, so maybe I'm just mixing it up with the ~~string~~ -> str (and similar) standardization. I did a quick glance and `pandas.DataFrame` is more the rule than exception, but I'll add a task to do an audit and try to track down any missing ones. (I do see one that I'll add on with the edit mentioned above.)